test/runner.py

```diff
@@ -606,6 +699,9 @@ def set_env(name, option_value):
 
   browser_common.init(options.force_browser_process_termination)
 
+  if options.log_test_environment:
```

Comment:
I guess we should probably print all this into in circle CI too.  The best way to do that is probably to add `or os.getenv('CI')`.